lib/components/narrative/default/access-leg.js

```diff
@@ -44,11 +48,17 @@ export default class AccessLeg extends Component {
           className='header'
           onClick={this._onLegClick}>
           <span><Icon type={`caret-${active ? 'down' : 'right'}`} /></span>
-          <span><b>{leg.mode}</b></span>
-          {' '}
-          <span className='leg-duration'>{coreUtils.time.formatDuration(leg.duration)}</span>
-          {' '}
-          <span className='leg-distance'>({humanizeDistanceString(leg.distance)})</span>
+          <FormattedMessage
+            id='components.AccessLeg.summary'
+            values={{
+              distance: humanizeDistanceString(leg.distance),
+              distanceSpan: contents => <span className='leg-distance'>{contents}</span>,
+              durationSpan: contents => <span className='leg-duration'>{contents}</span>,
```

Comment:
Remove duplicate line.

Reply:
Should I generalise the function to set the class name to 'leg-distance/duration'

Reply:
I don't think it is necessary.